test/jasmine/tests/pie_test.js

```diff
@@ -376,10 +356,37 @@ describe('Test event property of interactions on a pie plot:', function() {
     beforeEach(function() {
         gd = createGraphDiv();
         mockCopy = Lib.extendDeep({}, mock);
+        Lib.extendFlat(mockCopy.data[0], {
+            ids: ['marge', 'homer', 'bart', 'lisa', 'maggie'],
+            customdata: [{1: 2}, {3: 4}, {5: 6}, {7: 8}, {9: 10}]
```

Comment:
Nice. I was wondering if `customdata` would get passed properly :tada:

Reply:
Which by the way :white_check_mark: https://github.com/plotly/plotly.js/issues/2048